packages/terra-form-select/src/_Frame.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
-import { KeyCodes, Variants } from './_constants';
+import { KeyCode, Variants } from './_constants';
```

Comment:
I feel like we should directly import from the keycode package and remove the keycode exports from `_constants.js`